contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/links/SpringDataAssociationLinkCollector.java

```diff
@@ -47,18 +50,21 @@ public Links getLinksFor(Object object, Links existing) {
         Path selfPath = new Path(selfLink.expand().getHref());
 
         var links = new ArrayList<Link>();
-
-        entities.getRequiredPersistentEntity(object.getClass()).doWithAssociations(
-                (SimpleAssociationHandler) association -> {
-                    // We create a property here, so the name matches the one generated for the HAL-FORMS configuration
-                    // That will automatically create the HAL-FORMS options for the property
-                    var property = new JacksonBasedProperty(new DataRestBasedProperty(new PersistentPropertyProperty(association.getInverse())));
-                    for (Link link : associationLinks.getLinksFor(association, selfPath)) {
-                        var linkName = HalLinkRelation.of(link.getRel()).getLocalPart();
-                        var cgRelLink = link.withRel(ContentGridLinkRelations.RELATION).withName(linkName);
-                        links.add(addAssociationAffordance(cgRelLink, object.getClass(), property));
-                    }
-                });
+        var entity = entities.getRequiredPersistentEntity(object.getClass());
+        entity.doWithAssociations((SimpleAssociationHandler) association -> {
+            // We create a property here, so the name matches the one generated for the HAL-FORMS configuration
+            // That will automatically create the HAL-FORMS options for the property
+            var property = new JacksonBasedProperty(new DataRestBasedProperty(new PersistentPropertyProperty(association.getInverse())));
+            for (Link link : associationLinks.getLinksFor(association, selfPath)) {
+                var linkName = HalLinkRelation.of(link.getRel()).getLocalPart();
+                var cgRelLink = link
+                        .withRel(ContentGridLinkRelations.RELATION)
+                        .withName(linkName)
+                        .withTitle(resolver.resolve(LinkTitles
+                                .forProperty(entity.getType(), association.getInverse().getName())));
```

Comment:
This formatting looks off.

I would suggest formatting it like this (if it fits on the line then)
```suggestion
                        .withTitle(resolver.resolve(
                            LinkTitles.forProperty(entity.getType(), association.getInverse().getName())
                        ));
```